site/examples/examplesPageStyle.less

```diff
@@ -348,6 +348,12 @@
   }
 }
 
+.exactScrollTopControls {
+  display: flex;
+  justify-content: space-around;
+  align-items: baseline;
+}
```

Comment:
can you please explain what does this do

Reply:
This was a left over from some old test example. I removed those

Reply:
ok